examples/jsm/objects/Reflector.js

```diff
@@ -46,6 +46,8 @@ var Reflector = function ( geometry, options ) {
 
 	var textureMatrix = new Matrix4();
 	var virtualCamera = new PerspectiveCamera();
+	virtualCamera.layers.disable(0);
```

Comment:
Are you aware that disabling layer 0 will break all existing usage of `THREE.Reflector`?